web/yo/app/scripts/controllers/gene.js

```diff
@@ -2885,6 +2891,7 @@ angular.module('oncokbApp')
                     // This will only happen if the currentReviewer is not empty
                     $scope.fileEditable = false;
                 }
+                $scope.geneEditable = $scope.fileEditable;
```

Comment:
At line 2886, when the moment you change fileEditable, shouldn't it trigger the watcher to update geneEditable already?